### DISCUSSION TITLE
Random blank bio/status updates

### DISCUSSION THREAD

**Discussion by DireZombie** [vrcx-team/VRCX#933] [Q&A]
Not sure if this is the correct place to ask or discuss this as I also don't know if it is a bug or if me restoring the 'userdata' folder has caused some sort of issue. 
I was logged out of VRCX due to a password change, upon logging back into the same account all history is missing. 
I restored the VRCX 'userdata' folder with a back up copy of the data that was present before being logged out. This has resulted in seemingly random erroneous [blank bio and status](https://ibb.co/PwNBpYX) updates for two friends. 
When these are clicked, VRCX will "update" the current bio/status as if it's just been changed despite the bio and statuses having remained the same at all times. 

Not sure if this is a bug as I cannot reliably recreate the events as it does not happen for all friends.
![Screenshot 2024-10-09 061832](https://github.com/user-attachments/assets/5d16d7d2-3754-41f7-8b88-78c993fdaae4)

**Comment by Natsumi-sama** [comment]
Click the arrow on the left side to check if previous Bio/Status was blank or the same, in the past this would be caused by different VRC API endpoints returning slightly different text.

**Reply by DireZombie** [reply]
Clicking the arrow shows that the Bio/Status remains the same
![Screenshot 2024-10-09 071514](https://github.com/user-attachments/assets/60d48707-c3a2-4a4a-ac61-578a7d35554e)
Aside from reinstalling the application, which I assume would remove my 'history', are there any work arounds?

**Reply by Natsumi-sama** [reply]
This is a server side issue so there is very little we can do about it, you could try debug it yourself in the console `Ctrl + Shift + I` with `$app.debugWebSocket = true;` and watch for which events are returning empty Bios/Statuses. If there is a pattern to it we might be able to add checks to ignore broken data, because I cannot reproduce this issue myself I can't check for potential workarounds.

**Reply by DireZombie** [reply]
I'll check it out and see what I find. 
So far it seems to only be occurring with two users, will update with any further information.

**Reply by DireZombie** [reply]
The only additional information I can provide from the console at the moment (the people that are returning blank bios/statuses haven't logged on at the time of writing this) is a "404(Not Found)" Error when clicking on a users name within VRCX to view more information, The information is still shown yet the console throws out two of the same error. However, this does not occur with the other two users who keep returning blank bios/statuses. 

![Screenshot 205211](https://github.com/user-attachments/assets/c6574318-b698-4547-bac3-c7e94338d153)

If there is anymore updates I will reply. I'm not familiar with JavaScript or HTML so I apologise if the information is not of much use, if there is a way I can provide more useful information please let me know.

**Reply by Natsumi-sama** [reply]
Those errors are harmless, if you run the command from my last message you'll be able to monitor raw data as it comes in from VRChat.

**Reply by DireZombie** [reply]
I have already run the command, hence why I am able to see the errors. There was a 401 error which  is just credential related.
The bio/status updates, while appearing in VRCX, don't seem to be showing up in the console. Or at least not from what I can tell.

**Reply by Natsumi-sama** [reply]
The command is unrelated to error visibility and needs to be ran every time you start VRCX, you will know when the command it working because over time a lot of logs will show up in your console.

**Reply by DireZombie** [reply]
Is `$app.debugWebSocket = true;` meant to return to being false? I've closed the app a few times and restarted my PC yet the command remains in the 'true' state.

**Reply by Natsumi-sama** [reply]
Running `$app.debugWebSocket` will return it's current state, running `$app.debugWebSocket = true;` will also return it's current state, as true, because after running that command it is now set to true.
